R/estimateDiversity.R

```diff
@@ -299,8 +299,8 @@ setMethod("estimateDiversity", signature = c(x="TreeSummarizedExperiment"),
                 call. = FALSE)
         }
         
-        # If 'faith' is one of the indices
-        if( "faith" %in% index ){
```

Comment:
Use && instead of &.

Reply:
1. This if sentence is not checking the specific tree specified by user. This is just checking if there is some tree

is.null( rowTree(tse, tree_name) )